modules/lang-painless/src/test/java/org/elasticsearch/painless/DefBootstrapTests.java

```diff
@@ -30,14 +30,17 @@
 import org.elasticsearch.test.ESTestCase;
 
 public class DefBootstrapTests extends ESTestCase {
+    private final Definition definition = Definition.INSTANCE;
```

Comment:
I did this in the tests instead of using `Definition.INSTANCE` everywhere because I think we'll remove `Definition.INSTANCE` when we're done making context sensitive whitelists.